### DISCUSSION TITLE
Looks like an architectural problem?

### DISCUSSION THREAD

**Discussion by milkawa** [helix-editor/helix#7767] [Troubleshooting]
(
Linux localhost 4.4.192-perf-g2bcc393 #1 SMP PREEMPT Thu Jun 17 21:54:58 CST 2021 aarch64 Android
)
~ $ cd helix-23.05-aarch64-linux
~/helix-23.05-aarch64-linux $ ls
LICENSE  README.md  contrib  hx  runtime
~/helix-23.05-aarch64-linux $ ./hx
bash: ./hx: cannot execute: required file not found
🤔

**Comment by archseer** [comment]
You're probably missing some libraries, run ld on the binary.

**Reply by milkawa** [reply]
oh yeah,i am fixed it,thank you.

**Comment by archseer** [comment]
(Also kernel 4.4 is pretty ancient)
